deploy.yml

```diff
@@ -1,12 +1,13 @@
 ---
-- name: Configure and deploy pythondev apps
-  hosts: sirbot
+
+- name: Configure servers
```

Comment:
I agree.

But this way it make the `website` and `sirbot` play more easily reusable by other people. Most of the things we setup in `common` aren't directly related to `sirbot` or `website` but more to the basic configuration of our server.
My idea is for someone to be able to launch one play with his custom inventory without us messing his fail2ban configuration for example